paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/resources/ResourceSourceFile.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.paparazzi.internal.resources
+
+/**
+ * Ported from: [ResourceSourceFile.kt](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:resource-repository/main/java/com/android/resources/base/ResourceSourceFile.kt)
```

Comment:
```suggestion
 * Ported from: [ResourceSourceFile.kt](https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:resource-repository/main/java/com/android/resources/base/ResourceSourceFile.kt)
```

Can we use the actual HEAD SHA instead of mirror-goog-studio-main, since the latter changes with each commit?